package-lock.json

```diff
@@ -9,7 +9,7 @@
       "resolved": "https://registry.npmjs.org/accepts/-/accepts-1.3.5.tgz",
       "integrity": "sha1-63d99gEXI6OxTopywIBcjoZ0a9I=",
       "requires": {
-        "mime-types": "2.1.18",
+        "mime-types": "~2.1.18",
```

Comment:
I have no clue why this happens on my machine. `npm install` always updates packaeg-lock and moves dependency versions from exact to minor or patch level updatable ones.

```

$ node -v
v8.11.3

$ npm -v
6.4.1

```

Reply:
Search for `es6-symbol`. This goes from `3.1.1` to `~3.1.1` and `^3.1.1`

Reply:
For ref: https://github.com/npm/npm/issues/20891
https://github.com/npm/npm/issues/20434#issuecomment-392421024